src/components/button/__tests__/__snapshots__/Button.test.tsx.snap

```diff
@@ -4,6 +4,7 @@ exports[`components/button/Button should not render a RadarAnimation if showRada
 
 exports[`components/button/Button should render a RadarAnimation if showRadar is true 1`] = `
 <RadarAnimation
+  className=""
```

Comment:
Such changes in this file are due to the use of defaultProps over defaults in args

Reply:
Worth pointing out `defaultProps` is on the deprecation path, so it might be worth it to go the other way if there are no issues with it.

https://twitter.com/dan_abramov/status/1133878326358171650

Reply:
interesting, I guess saves some time then